File: solutions/beecrowd/1020/1020.clj
(ns main)

(defn main []
  (loop [line (read-line)]
    (when line
      (let [value (-> line
                      (Integer/parseInt))
            years (quot value 365)
            months (-> (mod value 365)
                       (quot 30))
            days (-> (mod value 365)
                     (mod 30))]
        (printf "%d ano(s)%n%d mes(es)%n%d dia(s)%n" years months days))
      (recur (read-line)))))

(main)

File: solutions/beecrowd/1020/1020.js
const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  terminal: false
});

(() => {
  rl.on('line', (line) => {
    let a = 0
    let m = 0
    let t = parseInt(line)

    if (t >= 365) {
        a = Math.floor(t / 365)
        t %= 365
    }

    if (t >= 30) {
        m = Math.floor(t / 30)
        t %= 30
    }

    console.log(`${a} ano(s)`)
    console.log(`${m} mes(es)`)
    console.log(`${t} dia(s)`)
  })
})()
